support-e2e/tests/genericCheckout.test.ts

```diff
@@ -0,0 +1,130 @@
+import { expect, test } from '@playwright/test';
+import { email, firstName, lastName } from './utils/users';
+import { setTestUserDetails } from './utils/testUserDetails';
+import { checkRecaptcha } from './utils/recaptcha';
+import { fillInCardDetails } from './utils/cardDetails';
+import { fillInDirectDebitDetails } from './utils/directDebitDetails';
+import { fillInPayPalDetails } from './utils/paypal';
+import { setupPage } from './utils/page';
+import { afterEachTasks } from './utils/afterEachTest';
+
+interface TestDetails {
+	product: 'Contribution' | 'SupporterPlus';
+	ratePlan: 'Monthly' | 'Annual';
+	paymentType?: 'Stripe' | 'DirectDebit' | 'PayPal';
+	price: number;
+	country?: 'UK' | 'US' | 'AU' | 'EU';
+}
+
+const glyphs: Record<'UK' | 'US' | 'AU' | 'EU', string> = {
```

Comment:
Why `string`?

Reply:
`string` looks like the recognised type used to display glyphs through support-frontend?